plat-stm32mp1: clock: few changes to prepare for scmi clock support

etienne-lms opened the pull request:
<!--
    If you are new to submitting pull requests to OP-TEE, then please have a
    look at the list below and tick them off before submitting the pull request.

    1. Read our contribution guidelines:
         https://optee.readthedocs.io/general/contribute.html

    2. Read the contribution section in Notice.md and pay extra attention to the
       "Developer Certificate of Origin" in the contribution guidelines.

    3. You should run checkpatch preferably before submitting the pull request.

    4. When everything has been reviewed, you will need to squash, rebase and
       add tags like `Reviewed-by`, `Acked-by`, `Tested-by` etc. More details
       about this can also be found on the link provided above.

    NOTE: This comment will not be shown in the pull request, so no harm keeping
    it, but feel free to remove it if you like.
-->

jforissier commented on core/arch/arm/plat-stm32mp1/drivers/stm32mp1_rcc.h:
```diff
@@ -530,6 +530,11 @@
+/* RCC_MP_TZAHB6EN(SET|CLR)R bit fields */
+#define RCC_MP_TZAHB6ENSETR_MDMA_POS		0
+#define RCC_MP_TZAHB6ENSETR_MDMA		\
+					BIT(RCC_MP_TZAHB6ENSETR_MDMA_POS)
```
Would look better with one less tab I suppose (I mean, remove one tab before the `0` and align `BIT(...` with the `0`).

etienne-lms commented:
Fixed indentation in commit "plat-stm32mp1: add mdma secure clock".
Fixed message of commit "plat-stm32mp1: clock: don't embed unused non-secure uart clocks".
Tag applied.
Thanks for the review and suggestions.
